static/test/javascripts/spec/common/adblock-banner.spec.js

```diff
@@ -0,0 +1,22 @@
+define([
+    'common/modules/commercial/adblock-banner-config',
+    'common/modules/adblock-banner'
+], function (adblockBannerConfig, AdblockBanner) {
+
+    describe('Adblock banner', function () {
+
+        var ukBanners;
+        beforeEach(function () {
+            document.innerHtml = '<div class="top-banner-ad-container--desktop"></div>';
+            ukBanners = adblockBannerConfig.getBanners('UK');
+        });
+
+        it('should be able to render every type of banner', function () {
```

Comment:
If you wish to test that the model variables are present, it may be preferable to just read them off the objects themselves. Testing the actual render means writing an async test and changing the contract of the code-under-examination to return a promise that informs the test when the fastdom write has finished.